lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
+        cube.add_dim_coord(DimCoord(standard_name='longitude',
+                                    units='degrees',
+                                    points=np.arange(3),), 1)
```

Comment:
@ajdawson From my perspective, the choice is arbitrary, so the decision to try var_name was purely aesthetic.  Given it was added as part of this fix - rather than modifying an existing test - I'm not sure I see the downside, but that's a bit of a moot point: I'll change it back momentarily.